### DISCUSSION TITLE
README for v3

### DISCUSSION THREAD

**Discussion by connecteev** [spatie/laravel-stripe-webhooks#161] [General]
I am going through the upgrade steps for v2 -> v3.

Does [this section](https://github.com/spatie/laravel-stripe-webhooks#performing-custom-logic) of the README.md need to be updated to be consistent with the [v3 upgrade guide](https://github.com/spatie/laravel-stripe-webhooks/blob/main/UPGRADING.md#config-file-changes)?

Am I understanding correctly that it should be something like this?
```
<?php

namespace App\Jobs;

use \Spatie\WebhookClient\Models\WebhookCall;

// Based on https://github.com/spatie/laravel-stripe-webhooks#performing-custom-logic
class MyCustomStripeWebhookJob extends WebhookCall
{
    /**
     * Execute the job.
     *
     * @return void
     */
    public function handle()
    {
        // do some custom stuff beforehand

        parent::handle();

        // do some custom stuff afterwards
    }
}

```

Also, are the rest of the references to ProcessStripeWebhookJob in the README correct?

**Comment by freekmurze** [comment]
On first glance, the readme is correct, you should indeed extend `ProcessStripeWebhookJob` which is present in the package.

In your code snippet, your letting your custom job extends a model, which does not make sense.

**Comment by connecteev** [comment]
Thanks for clarifying.
